x/ecocredit/client/testsuite/tx.go

```diff
@@ -1426,6 +1428,81 @@ func (s *IntegrationTestSuite) TestTxBuyDirect() {
 	}
 }
 
+func (s *IntegrationTestSuite) TestUpdateProjectMetadata() {
+	val0 := s.network.Validators[0]
+	valAddrStr := val0.Address.String()
+	clientCtx := val0.ClientCtx
+	cmd := coreclient.TxUpdateProjectMetadataCmd()
+	_, projectId, _ := s.createClassProjectBatch(clientCtx, valAddrStr)
+
+	unauthAcc, _, err := val0.ClientCtx.Keyring.NewMnemonic("unauthMetadata", keyring.English, sdk.FullFundraiserPath, keyring.DefaultBIP39Passphrase, hd.Secp256k1)
+	s.Require().NoError(err)
+	unauthAddr := unauthAcc.GetAddress()
```

Comment:
We can leverage the testsuite setup here and use `s.addr` rather than creating a new account.

```suggestion
	unauthAddr := s.addr
```

Reply:
fixed in https://github.com/regen-network/regen-ledger/pull/1105/commits/f15412e6d8561518a285494f1b1e12722a8f86fd